Logger/IEntity.cs

```diff
@@ -1,7 +1,7 @@
-﻿namespace Logger;
+namespace Logger;
 public interface IEntity
 {
+    Guid Id { get; init; }
```

Comment:
Can capitalize Id to ID
```suggestion
    Guid ID { get; init; }
```

Reply:
Unless I'm misremembering, C# is strictly Pascal-case, even for abbreviations (for example, JsonSerializer instead of JSONSerializer).

Reply:
The reasons we did it this way was because what @joshmerrell explained and that in the Storage file, the professor is using this property as "object.Id"